target/arduino/export.bat

```diff
@@ -0,0 +1,89 @@
+@ECHO off > nul
+:: Batch file to install BasicMAC from the git repository in to an
+:: existing Arduino installation by specifying the location of the 
+:: User library directory e.g.: "C:\Users\UserName\Documents\Arduino\libraries"
+
+:: \name Remko Welling (pe1mew@gmail.com)
+:: \date 8-7-2020
+:: \version 1.0
+
+:: \todo This script cannot work when whitespaces are in the directory path.
+:: As a workaround, export the library else where and move it in to the Arduino library directory.
```

Comment:
What part breaks exactly? Isn't this something that can be fixed with proper quoting? If not, can the script check for this and show an error message instead?

Reply:
I investigated on quoting but it didn't solve the issue. I will investigate on checking for spaces and throw an error. Unless some one comes with a more fundamental solution of coarse.

Reply:
The problem was caused by double quotes that are part of "value" of the value. 
I implemented a solution to remove double quotes from the variable in the batch file. 
The btach file now works with white spaces in the file directory name.